### DISCUSSION TITLE
SI-SNR loss function in ConvTasNet

### DISCUSSION THREAD

**Discussion by cemox35** [asteroid-team/asteroid#565] [Q&A] [answered]
Hi, I want to use SI-SNR loss, however, I cannot find it. Should I write my own loss or can it be possible to use it in an asteroid?

Edit: SI-SNR = SI-SDR? I looked at the asteroid SI-SDR implementation and SI-SNR formula. They are the same.

**Comment by mpariente** [comment, marked as answer]
Exactly, they are the same thing. It's just a naming convention.
